lib/demo_web/controllers/user_auth.ex

```diff
@@ -0,0 +1,139 @@
+defmodule DemoWeb.UserAuth do
+  import Plug.Conn
+  import Phoenix.Controller
+
+  alias Demo.Accounts
+  alias DemoWeb.Router.Helpers, as: Routes
+
+  # Make the remember me cookie valid for 60 days.
+  # If you want bump or reduce this value, also change
+  # the token expiry itself in UserToken.
+  @max_age 60 * 60 * 24 * 60
+  @remember_me_cookie "user_remember_me"
+  @remember_me_options [sign: true, max_age: @max_age]
```

Comment:
It may be worth pointing out that signed cookies as used here in `put_resp_cookie` and `fetch_cookies` requires Plug 1.10, which is brought in by the not-yet-released Phoenix 1.5. I mention this because it took me the better part of the day to figure out why the "remember me" functionality was silently failing (except once when Plug choked on the token's binary encoding).